README.md

```diff
@@ -77,18 +78,32 @@ drives the rest of the site.
 One other slightly unusual aspect of the frontend: normally, you'd have the `dist`
 directory ignored, as you don't want to commit generated files. In this case, though,
 we don't want `ember` to be a dependency of installing `rustdoc`, and so we do commit
-those generated files.
+those generated files. They are bundled in the `rustdoc-ember` binary.
+
+#### Alternative Frontends
+
+`rustdoc` allows using alternative frontends, provided that the frontend
+conforms to a particular interface.
+
+- The frontend must allow a single positional argument. This argument will be
+  the path to the documentation JSON generated by the backend.
```

Comment:
yeah, I wonder if maybe streaming over stdin isn't better. I feel like it's a tradeoff; my frontend would need to take that stdin and write it to a file anyway, whereas your implementation currently needs to read the file outputted.

I think I'm going to argue that it should be over stdin, because in that case, it only needs to write a file if it actually needs to write a file, otherwise we've left an extraneous file around. Thoughts?

Reply:
Yeah, I think stdin makes more sense. Certainly if we want to support streaming output.